### ISSUE TITLE
Declarative streaming API for cluster deployments

### ISSUE THREAD

**Issue by Neverlord** [actor-framework/actor-framework#572]
Streaming in CAF allows us to address many Big Data use cases. In this domain, however, CAF needs a good clustering story with:

- High-level DSL
- Automatic placement of individual actors
- Automatic snapshotting & re-deployment on error

**Comment by Neverlord** [comment]
On hold until we have clustering infrastructure in CAF.

**Comment by Neverlord** [comment]
Well, back in 2017 this seemed like a good idea. 🙂

Clustering did not really develop into a driver for CAF development. The old vision for the streaming API also never fully materialized as a result. CAF seems much more frequently used for Edge Computing, Micro Services or simply as concurrency toolkit. Hence, we shifted our focus to asynchronous building blocks, new networking facilities like WebSocket integrations and most recently composable flows that will now serve as backbone for re-implementing a much different streaming API.

Long story short: no longer relevant. Closing.
